x/commerciokyc/keeper/grpc_query_membership.go

```diff
@@ -3,24 +3,48 @@ package keeper
 import (
 	"context"
 
+	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkErr "github.com/cosmos/cosmos-sdk/types/errors"
 
 	"github.com/commercionetwork/commercionetwork/x/commerciokyc/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 func (k Keeper) Memberships(c context.Context, req *types.QueryMembershipsRequest) (*types.QueryMembershipsResponse, error) {
```

Comment:
The name of the method could mislead to think this returns all the memberships, but it returns only the valid ones. Consider renaming it (e.g. `ValidMemberships`) or to add documentation comments above the signature.